### ISSUE TITLE
Baseline UX + a11y review

### ISSUE THREAD

**Issue by drewbo** [cloud-gov/pages-editor#7]
Let's do an initial review of the demo app for egregious UX antipatterns or a11y violations

## Acceptance Criteria

- [ ] Determine next steps and acceptance criteria
